### PR TITLE
acrn-unit-test: add acrn readme copyright and sample test case

### PR DESCRIPTION
**Pull request by shenfang2019**
Signed-off-by: Shen Fangfang <fangfang.shen@intel.com>

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -1,3 +1,5 @@
+acrn-unit-test inherits from kvm-unit-tests and will keep kvm-unit-tests license.
```
Comment (gvancuts) [inline COPYRIGHT:1]: Should we add the Intel copyright nonetheless?

File: x86/Makefile.common [hunk 2]
```diff
@@ -85,7 +85,8 @@ tests-common = $(TEST_DIR)/vmexit.$(exe) $(TEST_DIR)/tsc.$(exe) \
                $(TEST_DIR)/tsx-ctrl.$(exe) \
                $(TEST_DIR)/eventinj.$(exe) \
                $(TEST_DIR)/smap.$(exe) \
-               $(TEST_DIR)/umip.$(exe)
+               $(TEST_DIR)/umip.$(exe) \
+	        $(TEST_DIR)/hello_acrn.$(exe)
```
Comment (gvancuts) [inline x86/Makefile.common]: Looks like the indentation is slightly off

File: COPYRIGHT (hunk 1 above)
Comment (shenfang2019) [reply COPYRIGHT:1]: thanks for reminder, could you help to share intel copyright with me? then I will add it.

File: x86/Makefile.common (hunk 2 above)
Comment (shenfang2019) [reply x86/Makefile.common]: yes, I will update.

File: COPYRIGHT (hunk 1 above)
Comment (gvancuts) [reply COPYRIGHT:1]: Here you go: `Copyright (C) 2022 Intel Corporation`

Comment (shenfang2019) [reply COPYRIGHT:1]: Thanks, done

File: x86/Makefile.common (hunk 2 above)
Comment (shenfang2019) [reply x86/Makefile.common]: Done

**Review by gvancuts — APPROVED** [review]

**Pull request merged**